moodle-scalable-cluster-ubuntu/azuredeploy.parameters.json

```diff
@@ -2,26 +2,11 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
         "parameters": {
-                "autoscaleVmCount":            { "value": 10 },
-                "autoscaleVmSku":              { "value": "Standard_DS2_v2" },
-                "azureBackupSwitch":           { "value": true },
-                "storageAccountType":          { "value": "Standard_LRS"},
-                "controllerVmSku":             { "value": "Standard_DS1_v2" },
-                "dbServerType":                { "value": "mysql" },
+                "redisDeploySwitch":           { "value": false },
```

Comment:
remove the non GEN values here and just use defaultValues in the template

Reply:
Actually, the default value change to `redisDeploySwitch` (or any other params) requires some internal discussion (@rgardler -- please chime in). We chose the default values in `azuredeploy.json` with some direction in mind, but here in the parameters file, we need to change it to minimize the test deployment time, so that's why it's changed here.

@rgardler , I'm in favor of changing this default to false in `azuredeploy.json`, but I wouldn't do that without your agreement first.

Reply:
I'm happy to follow your guidance on this one. My reading of the load testing results is that Reddis is not needed in anything but the most active scenarios. In which case I would support false by default.

Reply:
Thanks @rgardler . Updated the default, parameters.json, and pushed the commit.